src/isar/config/predefined_poses/predefined_poses.py

```diff
@@ -303,7 +297,7 @@
         frame=Frame.Asset,
     ),
     # EJE-342-1004.03 Robot orientation 90 degrees
-    "EJE-342-0226.10": Pose(
+    "EJE-342-1004.03": Pose(
```

Comment:
Why is this tag changed?

Reply:
The tag was changed to match the comment above. There was two entries with key "EJE-342-0226.10", so I assumed this one was a typo.

Reply:
That seams reasonable